Doc/using/cmdline.rst

```diff
@@ -1110,6 +1110,12 @@ conflict.
 
    .. versionadded:: 3.13
 
+.. envvar:: PY_COLORS
+
+   If this variable is set to ``1``, the interpreter will colorize error locations
+   when displaying tracebacks. Setting it to ``0`` deactivates this behavior.
```

Comment:
The name of this thing is pretty generic. Should it be documented in a less specific way in case it is used in the future for more than just error locations? Or maybe it should have a more specific name?

Reply:
Is the same variable that pytest uses, so I chose it to not have to reinvent something that people are already using:

https://github.com/pytest-dev/pytest/blob/022f1b4de546c8b3529e071965555888ecf01cb4/src/_pytest/_io/terminalwriter.py#L28

Reply:
> Should it be documented in a less specific way in case it is used in the future for more than just error locations?

I am happy to document this in a less specific way and refer to "colorize output" or something like that. Would that work?